### PR TITLE
cleaning up pac poc apps

### PR DESCRIPTION
**Pull request by galiri**

### CODE REVIEW

File: services.yaml [hunk 1]
```diff
@@ -173,11 +167,6 @@ services:
   version: v95
   count: 2
   sequentialDeployment: true
-- name: draft-content-rw-aurora-sidekick@.service
```
Comment (khatton-ft) [inline services.yaml]: The service / sidekick files for this service can also be removed.

File: services.yaml [hunk 2]
```diff
@@ -156,12 +156,6 @@ services:
 - name: content-rw-neo4j@.service
   version: 1.0.6
   count: 2
-- name: content-rw-s3-sidekick@.service
```
Comment (khatton-ft) [inline services.yaml]: The annotations-rw-rdb and annotations-rw-s3 services can also be removed from this file.

File: services.yaml (hunk 1 above)
Comment (galiri) [inline services.yaml]: done

File: services.yaml (hunk 2 above)
Comment (galiri) [inline services.yaml]: I cannae see them to remove??

**Pull request merged**